File: server/api/routes/gmc_feed_routes.py
from fastapi import APIRouter
from controllers.gmc_feed_controller import (
    generate_gmc_feed
)

router = APIRouter()

@router.get("/gmc-feed")
async def fetch_gmc_feed():
    return await generate_gmc_feed()

@router.get("/gmc-feed.xml")
async def fetch_gmc_feed_xml():
    return await generate_gmc_feed()

File: server/api/routes/gmcFeedRoutes.js
const express = require('express');
const { generateGMCFeed } = require('../controllers/gmcFeedController.js');

const router = express.Router();

// GET /api/gmc-feed.xml - Generate Google Merchant Center XML Feed
router.get('/gmc-feed.xml', generateGMCFeed);

module.exports = router;
